lib/CIGWServer.py

```diff
@@ -1,15 +1,13 @@
 #!/usr/bin/env python3
 import sys
-sys.path.append('lib')
-
-from os import path
 from io import open
 from json import dumps
 from flask import Flask, request
 from jenkins import Jenkins
 from controller.helpers import get_controllers
 
 
+sys.path.append('lib')
```

Comment:
Oh wait, nm.  This appends $CHARM_DIR/lib.  controller.helpers lives in pwd, which confused me because pwd is 'lib' too.